src/app/components/patterns/DemoCard.tsx

```diff
@@ -78,27 +68,29 @@ const useStyles = makeStyles((theme: Theme) =>
             color: theme.palette.text.primary,
         },
         demoCard: {
-            width: theme.spacing(35),
+            width: theme.spacing(38),
             maxWidth: '100%',
             margin: `0 auto`,
             padding: theme.spacing(1.5),
-            borderLeft: `${theme.spacing(1)}px solid ${theme.palette.secondary.main}`,
-            marginBottom: theme.spacing(4),
+            // borderLeft: `${theme.spacing(1)}px solid ${theme.palette.secondary.main}`,
```

Comment:
I think we need some way to make these cards pop off the page more...open to suggestions. When they're just white, they seem to really disappear on the page.